Makefile-docker

```diff
@@ -192,6 +192,7 @@ lint: ## lint the code
 	flake8 src/ services/ tests/
 	$(shell npm $(NPM_ARGS) bin)/prettier --check '**'
 	curlylint src/
+	black src/ services/ tests/
```

Comment:
this needs to be a command that would return the problems with the code, rather than auto-fix them.  I'd also be happier if it was at the top, before flake8.

Reply:
ah oops, right. there was `--check` before

Reply:
ah oops, right. there was `--check` before